### ISSUE TITLE
Initiating "New chat" causes crash [v14.1.1] 

### ISSUE THREAD

**Issue by fuglede** [guardianproject/ChatSecureAndroid#646]
When I try to initiate a new chat from my Jabber account, using the "+" icon above the buddy list, the app crashes.

**Comment by fuglede** [comment]
This one seems to have now changed. Using the "+" icon, I'm presented with a list of contacts. As soon as I scroll in this list, the app crashes.

**Comment by n8fr8** [comment]
Is this still happening? Is there anything unique about your contact list? size? strange avatars? names in other character sets?

**Comment by fuglede** [comment]
Yes, the problem persists and I'm back to crashing at the moment I click "+".
- Contact list is fairly short but does not fit on one screen.
- Not qutie sure what would constitute a strange avatar but there are none that are not simple images.
- Some non-ASCII characters; all within 8859-1 though.

**Comment by n8fr8** [comment]
There were some crashes we saw with an empty name for a contact. We'll have a new build soon to test.
